larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -427,18 +450,11 @@ void EvalRef(TfLiteContext* context, TfLiteNode* node,
   const auto* packed_filter = GetInput(context, node, 1);
   auto* output = GetOutput(context, node, 0);
 
-  // Using the standard TF Lite ConvParams struct.
-  // This requires extra step of converting the TfLiteBConv2DParams
-  // but unifies the interface with the default TF lite API for CONV params
-  // which is used in internal TF lite im2col functions.
-  ConvParams op_params;
-  GetConvParamsType(*params, op_params);
```

Comment:
I needed to access the `groups` attribute inside the reference kernel but couldn't because the built-in params don't have it, of course. I don't see any need to do this casting to the built-in params struct, so I've removed it and I now pass the bconv params struct directly.